app/views/home/index.html.slim

```diff
@@ -55,59 +54,71 @@
         = "Tim Peacock"
         = ' / '
         = link_to "cc", 'https://creativecommons.org/licenses/by/2.0/'
+  button.carousel-control-prev[type="button" data-bs-target="#myCarousel" data-bs-slide="prev"]
+    span.carousel-control-prev-icon[aria-hidden="true"]
+    span.visually-hidden
+      | Previous
```

Comment:
This string should be correctly localized and added to `config/locales/en.yml` (even if only used for screen readers).